src/shared/previewUtils.ts

```diff
@@ -58,38 +58,53 @@ export class PreviewUtils {
    * If it is in use then we increment the port number by 2 and check if it is in use or not.
    * This process is repeated until a port that is not in use is found.
    *
-   * @returns a port number to be used by the local dev server.
+   * @returns a pair of port numbers to be used by the local dev server for http and https.
    */
-  public static async getNextAvailablePort(): Promise<number> {
-    const userConfiguredPort = await ConfigUtils.getLocalDevServerPort();
+  public static async getNextAvailablePorts(): Promise<{ httpPort: number; httpsPort: number }> {
+    const userConfiguredPorts = await ConfigUtils.getLocalDevServerPorts();
 
-    if (userConfiguredPort) {
-      return Promise.resolve(userConfiguredPort);
+    if (userConfiguredPorts) {
+      return Promise.resolve(userConfiguredPorts);
     }
 
-    let port = LOCAL_DEV_SERVER_DEFAULT_PORT;
+    let httpPort = LOCAL_DEV_SERVER_DEFAULT_HTTP_PORT;
+    let httpsPort = httpPort + 1;
     let done = false;
 
     while (!done) {
-      const cmd =
-        process.platform === 'win32' ? `netstat -an | find "LISTENING" | find ":${port}"` : `lsof -i :${port}`;
+      const httpCmd =
+        process.platform === 'win32' ? `netstat -an | find "LISTENING" | find ":${httpPort}"` : `lsof -i :${httpPort}`;
+
+      const httpsCmd =
```

Comment:
This command is the same as the one above, with a different argument. Can we pull it out into a method?